lib/Doctrine/DBAL/Migrations/SqlFileWriter.php

```diff
@@ -77,12 +77,12 @@ public function write(array $queriesByVersion, $direction)
 
     private function buildMigrationFile(array $queriesByVersion, $direction)
     {
-        $string = sprintf("# Doctrine Migration File Generated on %s\n", date('Y-m-d H:i:s'));
+        $string = sprintf("# Doctrine Migration File Generated on %s" . PHP_EOL, date('Y-m-d H:i:s'));
```

Comment:
@stof As everyone is on probably on git and git take care of the line ending standardisation. Do you still think that it would be a problem ?